shapely/geos.py

```diff
@@ -21,12 +21,19 @@
 # Add message handler to this module's logger
 LOG = logging.getLogger(__name__)
 
+if sys.version_info[0] >= 3:
```

Comment:
We aren't planning to support Python 2.x in Shapely 1.8 so these changes aren't needed (and aren't wanted).

Reply:
I understand that the issue is larger than just us, but Cura runs 3.5 exclusively at the moment, so for us this isn't needed at all either.